bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/GarageDoorData.java

```diff
@@ -44,57 +41,47 @@
 	 * @param logData
 	 *            Boolean to determine if devicedata should be logged.
 	 */
-	public GarageDoorData(String deviceStatusData, boolean logData) {
+	public GarageDoorData(String deviceStatusData) {
 		ObjectMapper mapper = new ObjectMapper();
 		try {
 			JsonNode rootNode = mapper.readTree(deviceStatusData);
 			int ReturnCode = rootNode.get("ReturnCode").asInt();
-			logger.debug("myq ReturnCode: " + Integer.toString(ReturnCode));
+			logger.debug("myq ReturnCode: {}", Integer.toString(ReturnCode));
 
 			if (ReturnCode == 0) {
-				this.success = true;
 				if (rootNode.has("Devices")) {
 					JsonNode node = rootNode.get("Devices");
 					if (node.isArray()) {
-						if (logData) {
-							logger.info("Chamberlain MyQ Devices:");
-						}
-
+						logger.trace("Chamberlain MyQ Devices:");
 						int arraysize = node.size();
 						for (int i = 0; i < arraysize; i++) {
-							int deviceId = node.get(i).get("MyQDeviceId").asInt();
+							int deviceId = node.get(i).get("MyQDeviceId")
+									.asInt();
 							String deviceName = node.get(i).get("SerialNumber")
 									.asText();
 							String deviceType = node.get(i)
 									.get("MyQDeviceTypeName").asText();
 
-							if (deviceType.contains("Garage")
-									&& deviceType.contains("Door")
-									&& deviceType.contains("Opener")) {
+							if (deviceType.contains("GarageDoorOpener")) {
```

Comment:
@digitaldan This needs to be the old way as some openers DeviceTypeName are "GarageDoorOpener" other are "Garage Door Opener".
if (deviceType.contains("Garage") && deviceType.contains("Door") && deviceType.contains("Opener")) {}